v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -61,6 +61,10 @@ const (
 )
 
 func (o *ClusterResourcesGenerator) IDMS_ITMSGenerator(allRelatedImages []v2alpha1.CopyImageSchema, forceRepositoryScope bool) error {
+	if len(allRelatedImages) == 0 {
+		o.Log.Info("📄 Nothing mirrored. Skipping IDMS and ITMS files generation.")
```

Comment:
Instead of having a high level log message, can we have specific ones like "No digests found, skipping IDMS generation" as a else condition of the line 81? 

Same for the ITMS.